build/vanagon/components/ruby-2.6.rb

```diff
@@ -0,0 +1,27 @@
+component 'ruby-2.6' do |pkg, settings, platform|
```

Comment:
that's the ruby version we vendor, right? Our rubocop config is optimizied for 2.5? Is that on purpose?

Reply:
Not on purpose. I haven't messed with Rubocop in a long time, so I don't even know what would constitute optimization for ruby 2.6

Reply:
AFAIK all the 2.5 syntax works on 2.6. But maybe we should open an issue and later on update the syntax to match Ruby 2.6 in rubocop? Or should we stay on 2.5 since that's what Puppet6 vendors?

Reply:
I'd stay on 2.5 for now since the app is built on 2.5

We package with 2.6, but if someone pulls directly, we want the app to still work on what they expect.